### ISSUE TITLE
(Question) - Are windows 9x or 2000/xp taskbar add-ons compatible with RetroBar?

### ISSUE THREAD

**Issue by sammyrenard** [dremin/RetroBar#621]
For example there's old programs that replaced the clock with something livelier or fancier with different fonts and etc, and I wondered if these'd work here?

**Comment by yereverluvinunclebert** [comment]
No, they won't. Retrobar has nothing to do with the technology of the original taskbar and explorer in any version of windows. It is a recreation of the taskbar in basic functionality and visual appearance that is entirely independent of the o/s including explorer.

ie. it looks like Windows old taskbar and it attempts to offer a similar GUI but under the skin is it not compatible.

**Comment by TheWeirdo600** [comment]
It would be great if you were able to put, say, the WMP taskbar addon on Retrobar.
![image](https://github.com/dremin/RetroBar/assets/99914376/c7dabffa-7671-4afc-abf7-1470d2c6c34a)
I know nothing about code, but would "emulation" be possible here?
